app/models/ride.rb

```diff
@@ -202,7 +202,9 @@ def self.complete_status_values
   def self.confirm_scheduled_rides
     results = Hash.new(0)
     Ride.where(status: :scheduled).where('pickup_at < ?', SWITCH_TO_WAITING_ASSIGNMENT.minutes.from_now).each do |ride|
-      if ride.conversation && ride.ride_zone && !ride.ride_zone.bot_disabled
+      
+      # disabling the bot previously disabled confirmation, but I think we need those regardless...
+      if ride.conversation && ride.ride_zone # && !ride.ride_zone.bot_disabled
```

Comment:
Is there a reason you're keeping the commented-out code around? Is there a time when you anticipate wanting to bring it back? Under what conditions?

Reply:
Nope--kept temporarily to discuss then forgot to remove. Just did, thanks.